dotCMS/src/main/webapp/WEB-INF/velocity/static/content/content_form.vtl

```diff
@@ -0,0 +1,11 @@
+#set($hasForm = true)

```

Comment:
do you need change to 'content_form.vtl' here?:

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java#L497